PowerEditor/src/WinControls/WindowsDlg/WindowsDlg.cpp

```diff
@@ -865,6 +870,19 @@ void WindowsDlg::doClose()
 		}
 		ListView_SetItemCount(_hList, _idxMap.size());
 	}
+	doCount();
+}
+
+//this function will be called everytime when close is performed
+//as well as each time refresh is performed to keep updated
+void WindowsDlg::doCount()
+{
+	TCHAR count[32];
+	wsprintf(count, TEXT("%d"), _idxMap.size());
+	generic_string msg = TEXT("");
+	msg += TEXT("Total: ");

```

Comment:
`Total` should be made translatable.

Reply:
Updated the total. Now it should be translatable.